apps/core-app/src/main/modules/native-capabilities/index.test.ts

```
@@ -1,9 +1,15 @@
+import type { ModuleInitContext } from '@talex-touch/utils'
+import type { HandlerContext } from '@talex-touch/utils/transport/main'
+import type { TalexEvents } from '../../core/eventbus/touch-event'
 import { describe, expect, it, vi } from 'vitest'
 import { NativeEvents } from '@talex-touch/utils/transport/events'
```

ESLint

[error] Expected "`@talex-touch/utils/transport/events`" to come before "vitest".

(perfectionist/sort-imports)